tests/dragonfly/cluster_test.py

```diff
@@ -2580,3 +2580,68 @@ async def test_migration_timeout_on_sync(df_factory: DflyInstanceFactory, df_see
     await push_config(json.dumps(generate_config(nodes)), [node.admin_client for node in nodes])
 
     assert (await StaticSeeder.capture(nodes[1].client)) == start_capture
+
+
+@dfly_args({"proactor_threads": 4, "cluster_mode": "yes", "cache_mode": "true"})
+async def test_cache_cluster_data_migration(df_factory: DflyInstanceFactory, df_seeder_factory):
+    # Check data migration from one node to another with expiration and eviction
```

Comment:
you write with expiration but I dont see anything for expire keys here

Reply:
please add more description on this test. how you check the eviction flow. you took some decisions to use the non cluster client, this needs to be documented to explain why this decision was taken